### ISSUE TITLE
Cannot install Deepspeech using current instructions for Python 3.6.6 on Windows 10, possibly due to, "it is not compatible with this Python"

### ISSUE THREAD

**Issue by the-nose-knows** [mozilla/DeepSpeech#1521]
Current instructions are either incomplete or incorrect with the Windows Desktop environment; from the error output, it doesn't appear Windows is even supported due to a lack of windows WHLs, which would have been nice to know before I spent all this time trying to get my installation up and running. If this is the case, please please _please_ explicitly document that this is not available on Windows in a **conspicuous** location, like the main page on GitHub.

I installed Python 3.6.6 with all installation options enabled via the Custom Installation options. I couldn't install DeepSpeech so I updated pip from version 10.x to version 18.0

I was able to install virtualenv without issues, as well as start it up. However I still could not download deepspeech with pip, so I tried it again using increased verbosity. You can see the full text output of version-queries and the verbose request to install it at my gist here: https://gist.github.com/the-nose-knows/d06d6ad580125e8125ae994f7d12fbb3

**Comment by lissyx** [comment]
We never ever mentioned Windows support while we explicitely document macOS and Linux here: https://github.com/mozilla/DeepSpeech/blob/master/README.md

The release pages also explicitely list the supported platforms: https://github.com/mozilla/DeepSpeech/releases/tag/v0.1.1

**Comment by lissyx** [comment]
@the-nose-knows Also, some people on discourse reported being able to setup through the windows linux subsystem, if that can be of any help, yet it's very unlikely you can expect training to work on that kind of setup.

**Comment by lissyx** [comment]
@the-nose-knows Maybe you could send a PR expliciting that in the documentation? Obviously, it was obvious to us that Windows was not being supported for far, and thus we don't see **where** it's really important to state it for people.

**Comment by lissyx** [comment]
There's also issue #1123 open in case anyone might be interested in starting hacking on that. Back then, TensorFlow with Windows was not really a good thing, and our current focus is not yet really on that platform :)

**Comment by the-nose-knows** [comment]
@lissyx I don't see anywhere on the pages you linked to that mention the list of supported platforms. I see platforms that get mentioned for certain use-cases/scenarios, but that's it. I wouldn't assume a lack of Windows support for anything unless I was neck-deep in *nix territory already. That wasn't the case when I ran across DeepSpeech.

I'm taking a look at using Ubuntu via Windows Linux Subsystem. I'm having some issues with that, but I think it has more to do with SSL cert issues and little-to-nothing to do with DeepSpeech or the subsystem distro itself; I've already been able to install just about everything else using it after configuring my company's proxy. I'm hoping I can get past it, otherwise I'll pull out a Raspberry Pi 3 and take it from there.

I'll see about doing a documentation PR for you :)

**Comment by the-nose-knows** [comment]
Final Update: On Windows 10, I got deepspeech working on the Ubuntu distro available as a Windows Linux Subsystem on the Microsoft Store.

My primary points of friction was just setting all the env-vars needed for stuff, as it had been a couple years since I even touched a linux distro. First I needed to configure the proxy for apt-get, then the general env-vars used by many *nix apps, ie `http_proxy` and `https_proxy`

After that I was able to follow the existing documentation and got it working against one of the smoke_test files :)

```sh
username@DESKTOP-81FC02N:~/DeepSpeech$ deepspeech models/output_graph.pb data/smoke_test/LDC93S1.wav models/alphabet.tx
t models/lm.binary models/trie
Loading model from file models/output_graph.pb
2018-09-06 19:03:48.804771: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
Loaded model in 0.553s.
Loading language model from files models/lm.binary models/trie
Loaded language model in 0.948s.
Running inference.
she had a ducsuotangresywathorerall year
Inference took 6.056s for 2.925s audio file.
tosbanzai@DESKTOP-81FC02N:~/DeepSpeech$
```

When listening to that smoke_test file, I considered the fact that it got any of the words correctly a bonus. It wasn't very clear, so good job deepspeech! ^_^

**Comment by lissyx** [comment]
Closed by #1523
